CollectionView/UI/Cell/MyCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MyCollectionViewCell.swift
+//  CollectionView
+//
+//  Created by Felipe Gil on 2021-08-23.
+//
+
+import UIKit
+
+class MyCollectionViewCell: UICollectionViewCell {
+    @IBOutlet var imageView: UIImageView!
+    static let identifier = "MyCollectionViewCell"
+    var targetURL = ""
+    var task: DispatchWorkItem?
+ 
+    override func prepareForReuse() {
```

Comment:
Great job adding `prepareForReuse()`. It's always a good idea to check the documentation for the methods you're overriding. In this case, [`prepareForReuse` docs](https://developer.apple.com/documentation/uikit/uicollectionreusableview/1620141-prepareforreuse) say:

> The default implementation of this method does nothing. Subclasses such as `UICollectionViewCell` override this method and use it to perform relevant actions. So, if your subclass descends from `UICollectionViewCell` or another intermediate class, call super to ensure that your class gets the parent’s behavior.

So we're actually missing `super.prepareForReuse()` at the top of this method.